### PR TITLE
New landing page design

### PR DESCRIPTION
**Pull request by 0tuedon**

### CODE REVIEW

**Comment by 0tuedon** [comment]
https://transcription-review-frontend-bkchgaxbj-btc-knowledge-base.vercel.app/
Latest update here the youtube iframe still left and mobile responsiveness cc: @Extheoisah  @Emmanuel-Develops

**Review by Extheoisah — CHANGES_REQUESTED** [review]
First review pass on design only:
1. the font size is too large. we need to scale it to be normal according the to the screen aspect ratio.
2. there is an overflow of text in the cards in the 2nd section
image

3. the edges of the placeholder images are not rounded
image

4. the containers housing the images in these card sections do not have an overflow hidden property. the edges still overflow and show. Its not clean and neat as the design. pls look into it.
image

Aside these issues, this is a nice start👍. 
These are the obvious UI quirks i noticed. I'll leave another review on the code when these are fixed.

File: src/components/home/review/GreatTranscript.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        >
+          {" "}
+          Check out the following transcripts for inspiration:
+        </Text>
+        <UnorderedList
+          lineHeight={"150%"}
+          fontSize={{ base: "0.875rem", lg: "1.75rem" }}
+          letterSpacing={"0.78px"}
+        >
+          <ListItem>Silent Payments and Alternatives</ListItem>
```
Comment (adamjonas) [inline src/components/home/review/GreatTranscript.tsx:80]: These should be links I believe

**Review by adamjonas — COMMENTED** [review]
Wow. What a difference. Nice work. 

image

The mobile footer needs sizing and arrangement work

File: src/components/home/review/GreatTranscript.tsx (hunk 1 above)
Comment (0tuedon) [reply src/components/home/review/GreatTranscript.tsx:80]: oops, although i didn't get the links from the design.

**Comment by adamjonas** [comment]
Screenshot 2023-11-08 at 2 03 54 PM

This should be a link.

**Comment by adamjonas** [comment]
image

This image should crop out the admin checkboxes

**Comment by adamjonas** [comment]
Screenshot 2023-11-08 at 2 06 35 PM

another button that should be a link

**Comment by paperpsych** [comment]
Hi, here's my review on the design:

### The Hero Section

hero-rev


- We can make the wordmark bigger in size for better emphasis

Screenshot 2023-11-09 015710

- A smaller headline size here, similar to the size figma design and please format it in a manner such that it only uses 03 lines too like shown in Figma (lmk if the following snippet helps with the resizing)

Screenshot 2023-11-09 015426

- A smaller sub-headline size so that it fits in one screen with the headline and the other elements

Screenshot 2023-11-09 020009

- We should make the CTA buttons bigger too

Screenshot 2023-11-09 020537

- The faded orange grid in the background should have smaller grids

- And add the required animations to the CTAs and the video element

https://github.com/bitcointranscripts/transcription-review-front-end/assets/149538435/af29d119-1a9f-4c48-b171-f7066b590fd0

The CTAs (as mentioned above) are not links, they should be. I couldn't find a "copy link" option for them in the original site so I couldn't add them in the design. Here's the function of the CTAs in the hero section:

- "Get Started" is what takes the user to the Github Sign-In page that will ask them to connect with btctranscripts. The "Sign-In" in the nav bar of the original site does this.
- "Tell me More↓" scrolls down to the next section of the landing page. The "Get started" button does this on the original site.

Overall, the design has came out really well, I appreciate the effort that has been put into developing the site so far :)

P.S. Let me know if the code snippets are helpful or else I'll refrain from using them in the comments. Or else you can directly see them in Figma too !!

**Comment by Emmanuel-Develops** [comment]
Looks good!

Some border, radius and padding quirks 
Screenshot 2023-11-09 at 9 14 03 AM
Screenshot 2023-11-09 at 9 10 29 AM

**Comment by 0tuedon** [comment]
Working on this issues , Thanks @everyone

**Comment by adamjonas** [comment]
Getting a double header on mobile image

**Comment by adamjonas** [comment]
Also a double footer 

image

**Comment by 0tuedon** [comment]
> Getting a double header on mobile image

oh right fixing it.

**Comment by adamjonas** [comment]
The spacing and sizing in the footer needs adjustments. image

The top text seems totally disconnected from the bottom text.

**Comment by 0tuedon** [comment]
.

**Comment by adamjonas** [comment]
For mobile: 
1. Cut padding on "what you'll need" section
2. These elements have a vertical scroll in them on mobile: Screenshot 2023-11-10 at 3 44 21 PM
Screenshot 2023-11-10 at 3 43 59 PM
3. Font size consistency in step 3

@aassoiants does a review [here](https://drive.google.com/file/d/1j5XWv3HnyiNJkw4L2M8iDijxFpyi262E/view). Let's address the above pieces but I think we an wait on the others.

**Comment by 0tuedon** [comment]
> For mobile:
> 
> 1. Cut padding on "what you'll need" section
> 2. These elements have a vertical scroll in them on mobile:
> 
> 3. Font size consistency in step 3
> 
> @aassoiants does a review [here](https://drive.google.com/file/d/1j5XWv3HnyiNJkw4L2M8iDijxFpyi262E/view). Let's address the above pieces but I think we an wait on the others.



> For mobile:
> 
> 1. Cut padding on "what you'll need" section
> 2. These elements have a vertical scroll in them on mobile:
> 
> 
> @aassoiants does a review [here](https://drive.google.com/file/d/1j5XWv3HnyiNJkw4L2M8iDijxFpyi262E/view). Let's address the above pieces but I think we an wait on the others.

the lastest commit currently addresses these issues.

**Comment by aassoiants** [comment]
Thanks team!

**Review by Emmanuel-Develops — DISMISSED** [review]
All looks good!
rebase with updated dev branch and LGTM

**Comment by paperpsych** [comment]
Please reduce the size of the white boxes and the font size of the headings and body text inside the boxes.

Screenshot 2023-11-13 120209
Screenshot 2023-11-13 121224

File: src/components/home/CarouselCards.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        perPage: 1,
+      }}
+      style={{ maxWidth: "300px" }}
+    >
+      {stepContents.map((content) => (
+        <SplideSlide key={content.heading} style={{ width: "100%" }}>
+          <CarouseCardSlide {...content} />
+        </SplideSlide>
+      ))}
+      {/* <SplideSlide></SplideSlide> */}
```
Comment (Extheoisah) [inline src/components/home/CarouselCards.tsx:26]:
```suggestion
```
remove comment

File: src/components/home/NeedSection.tsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+              lineHeight={"105%"}
+              textAlign={"center"}
+              fontWeight={"semibold"}
+              display={{ base: "inline", lg: "none" }}
+            >
+              Well you’ll...
+            </Text>
+          </Flex>
+
+          {/*  */}
```
Comment (Extheoisah) [inline src/components/home/NeedSection.tsx:115]: remove
```suggestion
```

File: src/components/home/StepLayout.tsx [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+            as={"a"}
+            target="_blank"
+            href={link}
+            leftIcon={
+              <Box className="dark-wrapper">
+                <PlayIcon />
+              </Box>
+            }
+            fontSize={{ base: "12px", lg: "18px", "2xl": "24px" }}
+            // size={{ base: "md", lg: "lg" }}
```
Comment (Extheoisah) [inline src/components/home/StepLayout.tsx:88]:
```suggestion
```

File: src/components/home/steps/Step1.tsx [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+      desc2:
+        "In simple English, it means that BTCTranscripts will write your transcript onto GitHub so that it can be reviewed.",
+    },
+  ];
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setCurrentIndex((prevIndex) =>
+        prevIndex === images.length - 1 ? 0 : prevIndex + 1
+      );
+    }, 10000); // Change slide every 3 seconds (3000 milliseconds)
```
Comment (Extheoisah) [inline src/components/home/steps/Step1.tsx:35]:
```suggestion
    }, 10000); // Change slide every 10 seconds
```

File: src/components/home/steps/Step1.tsx [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+  });
+  return (
+    <StepLayout
+      stepNumber={1}
+      heading="Claim a recording to review"
+      sub="TLDR; Connect your GitHub to BTCtranscripts"
+      link={"https://www.youtube.com/watch?v=YNIFm0QFAuA&t=2m16s"}
+    >
+      <Flex flexDir={"column"} gap={10}>
+        {/* <Flex></Flex> */}
```
Comment (Extheoisah) [inline src/components/home/steps/Step1.tsx:49]:
```suggestion
```

File: src/components/home/steps/Step3.tsx [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+      desc: "If you want to see your transcript, visit your profile and click the link. On the transcript’s GitHub PR, you can click the “Files changed” nav button to view.",
+    },
+  ];
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setCurrentIndex((prevIndex) =>
+        prevIndex === images.length - 1 ? 0 : prevIndex + 1
+      );
+    }, 10000); // Change slide every 3 seconds (3000 milliseconds)
```
Comment (Extheoisah) [inline src/components/home/steps/Step3.tsx:34]:
```suggestion
    }, 10000); // Change slide every 10 seconds
```

**Review by Extheoisah — CHANGES_REQUESTED** [review]

**Review by Extheoisah — APPROVED** [review]

**Comment by paperpsych** [comment]
We'll need the sizing of text in this whole section to be reduced too.

Screenshot (926)
Screenshot (927)

---

Also, please change the orange on the whole website from "FFA500" to "F7931A".

---

Here the 2nd half of the "Authorize" step is staying dormant.

Screenshot 2023-11-14 185510

This is how it should look. And please adjust the vertical gap between the heading and the body to match the height & alignment of all the 03 parts of texts.

Screenshot 2023-11-14 185437

---
The "Watch tutorial" buttons should have more curve radius and more padding on top and bottom.

Screenshot 2023-11-14 190135
Screenshot 2023-11-14 190156

**Comment by paperpsych** [comment]
[Here's the link to the prototype for reference](https://www.figma.com/proto/PK9XQob6bVMw5Q0rJhYbu2/BTC-Transcript-Review?page-id=0%3A1&type=design&node-id=155-98&viewport=-1243%2C281%2C0.52&t=8Cmq8zkS9KibC809-1&scaling=scale-down-width&mode=design)

**Hero Section**

- Wordmark (logo text) slightly bigger
- Subheadline smaller in size
- Smaller YouTube video element
- Buttons seem to have unequal top and bottom padding
- "Tell me more" button's arrow needs to be smaller to match the button text's height

Screenshot (929)

---

**Overall text formatting on the website**

- Please match the text's line breaks
- Match the spacing of elements from the edge and other elements
- Match button hover animations

---

**Why edit transcripts?**

- Smaller font sizes
- The image's size needs to more rectangular instead of square
- And more gap from the left edge and less gap between the image and the text content

Screenshot (930)

---

**How it works**

- Text alignments

Group 4669

- Step 03's icon should be changed to the tick mark one
- "After Submission" text block should be of the PolySans typeface

---

**Review Process Tips**

- Smaller overall size of the section
- [Use this image](https://drive.google.com/file/d/1voXTa0ALPVcT0KNBKLEq3SOvFxpgOByP/view?usp=drive_link) for the split screen section

---

**Primary CTA and Footer**

- Changes in CTA button's size, accompanying copy's font size
- Gap between the primary CTA section, the review process section, and footer
- _Submit feedback_ side in Aeonik Fono typeface and _Bitcoin Dev Project_ side in PolySans typeface (and both need to be closer to their edges)

Screenshot (931)

Screenshot (932)

**Comment by 0tuedon** [comment]
> We'll need the sizing of text in this whole section to be reduced too.
> 

> Screenshot 2023-11-14 190135 Screenshot 2023-11-14 190156

All this issues is being referenced in #224

**Pull request merged**